### PR TITLE
[WIP] Quartz Boundary Conditions

### PR DESCRIPTION
**Pull request by Miracle-Ein**
I have implemented quartz boundary conditions for WarpX.

### CODE REVIEW

File: Examples/Tests/quartz_vs_pec_comparison/README_physical_verification.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# 石英边界条件物理验证指南
```
Comment (ax3l) [inline Examples/Tests/quartz_vs_pec_comparison/README_physical_verification.md:1]: Hi @Miracle-Ein, appreciate the readme!

Can you please add all documentation in English for this project? That is the language all our international contributors use.

File: README.md [hunk 2]
```diff
@@ -1,69 +1,236 @@
...
 [![Language: C++17](https://img.shields.io/badge/language-C%2B%2B17-orange.svg)](https://isocpp.org/)
-[![Language: Python](https://img.shields.io/badge/language-Python-orange.svg)](https://python.org/)  
-[![License WarpX](https://img.shields.io/badge/license-BSD--3--Clause--LBNL-blue.svg)](https://spdx.org/licenses/BSD-3-Clause-LBNL.html)
-[![DOI (source)](https://img.shields.io/badge/DOI%20(source)-10.5281/zenodo.4571577-blue.svg)](https://doi.org/10.5281/zenodo.4571577)
-[![DOI (paper)](https://img.shields.io/badge/DOI%20(paper)-10.1109/SC41404.2022.00008-blue.svg)](https://doi.org/10.1109/SC41404.2022.00008)
+[![Platform](https://img.shields.io/badge/platform-Linux%20|%20macOS%20|%20Windows-blue)](https://warpx.readthedocs.io)
+
+## 概述
+
+Lanzi-WarpX-Etching-Module 是基于 [WarpX](https://warpx.readthedocs.io) 的等离子体刻蚀模拟扩展模块，专门用于电容耦合等离子体(CCP)设备的数值模拟。本项目在WarpX的基础上添加了石英边界条件模块，能够准确模拟等离子体刻蚀过程中的绝缘材料效应。
```
Comment (ax3l) [inline README.md:10]: Please do not modify the main README file of the code you contribute to :)

File: Examples/Physics_applications/beam_beam_collision/warpx_used_inputs.txt [hunk 3]
```diff
@@ -0,0 +1,217 @@
+ColliderRelevant_beam1_beam2.intervals = every_red
```
Comment (ax3l) [inline Examples/Physics_applications/beam_beam_collision/warpx_used_inputs.txt:1]: Looks like an unrelated example was changed.

**Comment by ax3l** [comment]
Hi @Miracle-Ein,

Thank you for your first PR!

There are a few issues with it, which please need to be addressed if you like to include your new boundary conditions upstream:
- please do not change unrelated code, examples and documentation
- please add your new examples/tests in English language
- please add in the PR description, example README and in-line documentation references to the models (e.g, papers) you implement for the boundary conditions.
- please [follow our contribution guide](https://warpx.readthedocs.io/en/latest/developers/contributing.html#submit-a-pull-request).

At the moment, it looks a bit like this PR is fully automated from a LLM code generator. Please respond here if you like to update it to bring it to the quality we need to merge contributions.

**Comment by Miracle-Ein** [comment]
Hi,ax3l


Thank you very much for your valuable feedback.


As this is my first time contributing to an open-source project, I’m still learning the processes and best practices. I will take some time to carefully review your suggestions and revise the code accordingly.


I appreciate your patience and understanding as I work through these improvements. Please feel free to share any further suggestions or resources you think might help—your input is truly appreciated.




943575436
***@***.***

**Comment by ax3l** [comment]
Thanks!

I saw you opened a new PR with a cleaner implementation in #6056. Let us continue there. Closing this one here.

To update a PR, please just add a new commit to the branch that the pull request "pulls" from, no need to open a new PR for a suggested change.
